js/core/utils/ajax.js

```diff
@@ -163,7 +163,6 @@ var isCrossDomain = function(url) {
 
     try {
         urlAnchor.href = url;
-        urlAnchor.href = urlAnchor.href;
```

Comment:
I was interested in how such a code could be written. So I have made a small investigation.
https://github.com/DevExpress/DevExtreme/pull/1177/files#diff-5f87e0d504be9bdc9d62bcf19d73f6e8R150

Reply:
Are you ready for magic?
```
var urlAnchor = document.createElement("a");
urlAnchor.href = "12";
// urlAnchor.href = urlAnchor.href;
console.log(urlAnchor.host);
```
Run this code with comment, and without it in my 'favorite' browser (IE11)

Reply:
OMG. It's the craziest thing I have ever seen. If so we need to left some note at least.

Reply:
I agree. I have tested 3 browsers before my memory come back.

Reply:
Thank you @babich-a! I will fix this.